test/integration/Config.ts

```diff
@@ -12,12 +12,12 @@ export async function instantiateFromConfig(componentUrl: string, configFile: st
   variables?: Record<string, any>): Promise<any> {
   // Initialize the Components.js loader
   const mainModulePath = joinFilePath(__dirname, '../../');
-  const loader = new Loader({ mainModulePath });
-  await loader.registerAvailableModuleResources();
+  const manager = await ComponentsManager.build({ mainModulePath, logLevel: 'error' });
 
   // Instantiate the component from the config
   const configPath = toSystemFilePath(joinFilePath(__dirname, 'config', configFile));
-  return loader.instantiateFromUrl(componentUrl, configPath, undefined, { variables });
+  await manager.configRegistry.register(configPath);
```

Comment:
Can we reuse this part in order to speed up integration tests?

Reply:
The only thing that's safe to reuse is the `IModuleState` (can be manually build with [this](https://github.com/LinkedSoftwareDependencies/Components.js/blob/master/lib/loading/ModuleStateBuilder.ts#L24)).
Such a module state can be passed into the manager builder, so that it won't rebuild itself.
The generation of this module state should be most significant cause of slowdown, so doing this should make things a lot faster.

Reply:
Great! Could you please add that in a future PR?